caas/kubernetes/provider/k8s.go

```diff
@@ -179,6 +182,74 @@ func (k *kubernetesClient) PrepareForBootstrap(ctx environs.BootstrapContext) er
 	return nil
 }
 
+func newLabelRequirements(rs ...requirement) k8slabels.Selector {
+	s := k8slabels.NewSelector()
+	for _, r := range rs {
+		l, err := k8slabels.NewRequirement(r.key, r.operator, r.strValues)
+		if err != nil {
+			panic(errors.Annotate(err, "fix this"))
```

Comment:
yikes!

Reply:
moved to init.go to fail it earlier if it' has invalid conf. and bettered error msg.